src/authStrategies/RemoteAuth.js

```diff
@@ -0,0 +1,195 @@
+'use strict';
+
+const path = require('path');
+const fs = require('fs-extra');
```

Comment:
We need some try-catch since this modules are optional

`try {
   const fs = require('fs-extra');
   const unzipper = require('unzipper');
const archiver = require('archiver');
} catch {
    console.log('You don\'t have the optional packages, install .......')
    process.exit(1);
}`

Reply:
> Done